docs/google/devices.md

```diff
@@ -24,8 +24,179 @@ Several factors can influence this limit, but this can be estimated at about 40
+)
+```
+
+This circuit will execute the two gates in serial:
+
+
+```
+cirq.Circuit(
+  cirq.Moment(cirq.X(cirq.GridQubit(4,4))),
+  cirq.Moment(cirq.X(cirq.GridQubit(4,5)))
+)
+```
+
+Lastly, this circuit will only execute one gate, since the first gate
+is virtual and its moment will disappear:
+
+```
+cirq.Circuit(
+  cirq.Moment(cirq.Z(cirq.GridQubit(4,4))),
+  cirq.Moment(cirq.X(cirq.GridQubit(4,5)))
+)
+```
+
+## Gates supported
+
+The following lists the gates supported by Google devices.
+Please note that gate durations are subject to change as hardware isxi
+updated and modified, so please refer to the
+[device specification](https://cirq.readthedocs.io/en/latest/docs/google/specification.html)
+to get up-to-date information on supported gates and durations for
+specific processors
+
+### One qubit gates
+
+Google devices support arbitrary one-qubit gates of any rotation.
+The full complement of these rotations can be accessed by using the
+`cirq.PhasedXZGate`.  More restrictive one-qubit gates, such as
+the Pauli gates `cirq.X`, `cirq.Y`, `cirq.Z`, as well as the
+gate `cirq.PhasedXPowGate` can also be natively executed.
+
+#### Virtual Z gates
+
+Rotation around the Z axis is generally not a hardware operation.
+Instead, the compilation keeps track of the Z phase rotations,
+commuting them forward through the circuit until a non-commuting
+gate is reached.  This compilation is handled automatically for you.
+
+What this means is that `cirq.Z` and `cirq.ZPowGate` gates will
+have zero duration on the device.  Any moments containing only
+these gates will silently disappear from the circuit.
+
+#### Physical Z gates
+
+While most applications prefer shorter circuit durations and
+virtual Z gates, Google hardware does offer the possibility of
+applying a physical Z gate that performs a hardware operation
+to affect the frequency of the qubit.
+
+This can be done by applying a PhysicalZTag to the Z gate,
+such as in the following example:
+
+```
+cirq.Z(cirq.GridQubit(5, 5)).with_tags(cirq.google.PhysicalZTag())
+```
+
+### Two Qubit Gates
+
+Google devices provide several options for two-qubit gates.
+The availability of these gates is controlled by the gateset
+parameter that is used.
+
+Note that current hardware gates are noisy and not homogenous
+across the device.  Unitaries provided below are for the ideal case.
+Real unitaries applied to the qubit may vary from qubit to qubit
+and may drift over time.
+
+#### Sycamore Gate
+
+The hardware provides a gate known as the Sycamore gate that can
+be accessed using `cirq.google.SYC`.  This gate is equivalent to
+an FSimGate(π/2, π/6).  That is, it does both a partial swap and
+controlled phase rotation of the |11⟩ state.
+
+The unitary of this gate, which can also be found via the `cirq.unitary`
+function, is:
+
+$$
+\left[
+\begin{matrix}
+1 & 0 & 0 & 0 \\
+0 & 0 & -i & 0 \\
+0 & -i & 0 & 0 \\
+0 & 0 & 0 & e^{-i \pi \over 6}
```

Comment:
slight pref for `e^{-i \frac{\pi}{6}}.  Puts the `-i` out front instead of the minus in the numerator, and also LaTeX people yell at me for using `\over`.

Reply:
Sure.  Done.